<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGui</class>
 <widget class="QMainWindow" name="MainGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CometOS Packet Analyzer v0.2</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frameFilterMarker">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="frameFilterMarkerLabel">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="labelFilter">
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelMarker">
            <property name="text">
             <string>Marker</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameFilterMarkerEdit">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLineEdit" name="editFilter"/>
          </item>
          <item>
           <widget class="QLineEdit" name="editMarker"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="framePkts">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="framePktsContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>802</width>
         <height>380</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QTextBrowser" name="textFill"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frameControl">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="buttonLoad">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonExport">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Export</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCounter">
         <property name="text">
          <string>Packet Counter:  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="counter">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkHex">
         <property name="text">
          <string>Hex</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkDec">
         <property name="text">
          <string>Dec</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkAscii">
         <property name="text">
          <string>Ascii</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
